src/modules/esl-popup/core/esl-popup.ts

```diff
@@ -111,6 +110,16 @@ export class ESLPopup extends ESLToggleable {
   @attr({parser: parseBoolean, serializer: toBooleanAttribute, defaultValue: true})
   public override closeOnOutsideAction: boolean;
 
+  /**
+   * Focus behavior. Available values:
+   * - 'none' - no focus management
+   * - 'grab' - focus on the first focusable element
+   * - 'chain' (default) - focus on the first focusable element first and return focus to the activator after the last focusable element
```

Comment:
'grab' is like 'chain' but without hiding toggleable, isn't it?

Reply:
It's actually like autofocus without an ending handler so it toggleable grabs focus but does but does not handle focus loss or change focus order (including the focusable after last in toggleable scope)